browser/src/control/Control.RowHeader.ts

```diff
@@ -122,9 +122,9 @@ export class RowHeader extends cool.Header {
 		let selectionBackgroundGradient = null;
 		if (highlight) {
 			selectionBackgroundGradient = this.context.createLinearGradient(0, startY, 0, startY + entry.size);
-			selectionBackgroundGradient.addColorStop(0, this._selectionBackgroundGradient[0]);
```

Comment:
Can we please remove these
`this._selectionBackgroundGradient -> app.selectionBackgroundGradient` changes?

I am concerned about the change here. Please let me know if these are related to your work directly. If they are not, let's consider these for a different PR?